plugins/woocommerce-admin/client/two-column-tasks/task-list-completion-slot/utils.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../../utils';
+
+export const WC_TASK_LIST_COMPLETION_SLOT_NAME =
+	'woocommerce_task_list-completion_item';
+/**
+ * Create a Fill for extensions to add items to the WooCommerce Admin Task List completion component slot.
+ *
+ * @slotFill WooTaskListCompletionItem
+ * @scope woocommerce-admin
+ * @example
+ * const MyTasklistCompletionItem = () => (
+ * <WooTaskListCompletionItem>My Task List completion item</WooTaskListCompletionItem>
+ * );
+ *
+ * registerPlugin( 'my-extension', {
+ * render: MyTasklistCompletionItem,
+ * scope: 'woocommerce-admin',
+ * } );
+ * @param {Object} param0
+ * @param {Array}  param0.children - Node children.
+ * @param {Array}  param0.order    - Node order.
+ */
+export const WooTaskListCompletionItem: React.FC< { order?: number } > & {
+	Slot: React.FC< Slot.Props >;
```

Comment:
`React.FC` is going to be removed soon: https://github.com/woocommerce/woocommerce/pull/36405#issuecomment-1381184696. I think we can preemptively not use it

Reply:
Huh they're removing the only reason I use React.FC, which is the implicit type for `children: ReactNode`. 🤷‍♂️